src/types.ts

```diff
@@ -884,6 +889,17 @@ export type QueryMembersOptions = {
   user_id_lte?: string;
 };
 
+export type ReactivateUserOptions = {
+  created_by_id?: string;
+  name?: string;
```

Comment:
we don't support name in batch version?

Reply:
we don't, might be easier to handle batches where there is no conflict, and handle conflict using the simple endpoint. I don't think it will be a common scenario. Can be upgraded later if needed.

Reply:
agreed